code/modules/food_and_drinks/restaurant/customers/_customer.dm

```diff
@@ -286,3 +286,27 @@
 	first_warning_line = "Amigo! Don't touch me like that."
 	second_warning_line = "Compadre, enough is enough! Last warning!"
 	self_defense_line = "Time for you to find out what kind of robot I am, eh?"
+
+///MALFUNCTIONING - only shows up once per venue, very rare
+/datum/customer_data/malfunction
+	nationality = "Malfunctioning"
+	base_icon = "defect"
+	prefix_file = "strings/names/malf_prefix.txt"
+	speech_sound = 'sound/effects/clang.ogg'
+	clothing_sets = list("defect_wires", "defect_bad_takes")
+	is_unique = TRUE
+	orderable_objects = list(
+	/datum/venue/restaurant = list(/obj/item/toy/crayon/red = 1, /obj/item/toy/crayon/orange = 1, /obj/item/toy/crayon/yellow = 1, /obj/item/toy/crayon/green = 1, /obj/item/toy/crayon/blue = 1, /obj/item/toy/crayon/purple = 1),
+	/datum/venue/bar = list(/datum/reagent/consumable/failed_reaction = 1, /datum/reagent/spraytan = 1, /datum/reagent/reaction_agent/basic_buffer = 1, /datum/reagent/reaction_agent/acidic_buffer = 1))
```

Comment:
Space these out so that we don't get conflicts and ugly blames and ugly diffs whenever someone dares to edit them.